api/src/main/java/org/eclipse/microprofile/config/spi/ConfigSource.java

```diff
@@ -20,18 +20,18 @@
 import java.util.Map;
 
 /**
- * <p> Represent a config source, which provides properties. The config source includes: properties, xml, json files or datastore. <p>
+ * <p> Represent a config source, which provides properties. The config source can encapsulate: properties, xml, json files or datastore data. <p>
  * The default config sources:
  * <ol>
  * <li>System properties (ordinal=400)</li>
  * <li>Environment properties (ordinal=300)</li>
  * <li>/META-INF/microprofile-config.properties (ordinal=100)</li>
  * </ol>
  * 
- * <p>A ConfigSource will get picked up via the
- * {@link java.util.ServiceLoader} mechanism and must get registered via
+ * <p>ConfigSource will get picked up via the
+ * {@link java.util.ServiceLoader} mechanism and and can be registered via
  * META-INF/services/javax.config.spi.ConfigSource</p>
- * The other custom config source can be added programmatically via {@link org.eclipse.microprofile.config.ConfigProvider}.
+ * Other custom config source can be added programmatically via {@link org.eclipse.microprofile.config.ConfigProvider}.
```

Comment:
No, it is possible. The calling of ConfigProvider.getConfig() should trigger the builder to add default configure sources and autodiscovered configure sources. Otherwise, the ConfigProvider.getConfig is not useful. Thoughts? We should clarify this.